### ISSUE TITLE
Adjust tab flow in new design

### ISSUE THREAD

**Issue by klappradla** [bitcrowd/tickety-tick#231]
In the new compact design introduced in #221, the tabing workflow feels a bit off for cases where only a single ticket is matched _(which may assumed to be the default case for most trackers)_: the disabled `select` field still gets the focus on the 3rd press of the <kbd>tab</kbd> key.

Firefox however does not indicate the focus. Plus there is not really such a huge point in focussing the disabled field 🤔 . Still, people may get the impression that the popup lost focus all together, while in fact another press of the <kbd>tab</kbd> key would just bring them back.

See the two scenarios for a comparison:

#### Multiple Tickets

<img width="298" alt="Screenshot 2020-04-29 at 14 53 30" src="https://user-images.githubusercontent.com/3491815/80610460-b0129c00-8a39-11ea-81b1-cc7f44f60c3a.png">

#### Single Ticket

<img width="295" alt="Screenshot 2020-04-29 at 14 53 42" src="https://user-images.githubusercontent.com/3491815/80610485-b739aa00-8a39-11ea-8b8e-df19c7fefec1.png">

One way to make this more intuitive could be to not make the `select` tabable at all when it's disabled? What do you think?

**Comment by pmeinhardt** [comment]
Oh wow. Firefox tab-selects disabled elements. Didn't expect that. 👀 Thanks @klappradla ❤️

**Comment by klappradla** [comment]
Experimenting around with this a bit, it actually seems to be a false alarm 🙈.

The focus is not on the disabled select, but on either the whole popup or outside (I cannot really debug the HTML of the popup) which is still the same behavior as for the currently released version.

**TL;DR** the disabled select input is not tab-selected 👍

**Comment by klappradla** [comment]
FYI: the active element is the `body` then 🤷
